index.html

```diff
@@ -1,70 +1,443 @@
+            </div>
+            <div class="item4"></div>
+            <div class="item5">
+              <p class="year">2015</p>
+            </div>
+          </div>
+          <div class="text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="tags">
+            <ul class="ul">
+              <li class="html">
+                <p class="h">HTML</p>
+              </li>
+              <li class="css">
+                <p class="c">CSS</p>
+              </li>
+              <li class="js">
+                <p class="j">JavaScript</p>
+              </li>
+            </ul>
+          </div>
+
+          <div class="action">
+            <div class="boxx">
+              <button class="see-btn">
+                <div class="see">
+                  <p class="se">See Project</p>
+                </div>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="footer">
+      <div class="top">
+        <div class="about">
+          <h1>About Myself</h1>
+        </div>
+      </div>
+      <div class="content">
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+      </div>
+      <div class="footerbox">
+        <div class="let">
+          <h2 class="lets">LET’S CONNECT</h2>
+        </div>
+      </div>
+      <section class="social2">
+        <ul class="social-icons-set">
+          <li class="social-icon">
+            <a href="#"
+              ><i
+                class="fa-brands fa-github"
+                target="_blank"
+                aria-label="GitHub"
+              ></i
+            ></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"
+              ><i
+                class="fa-brands fa-linkedin-in"
+                target="_blank"
+                aria-label="Linkedin"
+              ></i
+            ></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"
+              ><i
+                class="fa-brands fa-angellist"
+                target="_blank"
+                aria-label="Angellist"
+              ></i
+            ></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"
+              ><i
+                class="fa-brands fa-twitter"
+                target="_blank"
+                aria-label="Twitter"
+              ></i
+            ></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"
+              ><i class="fa-solid fa-m" target="_blank" aria-label="Medium"></i
+            ></a>
+          </li>
+        </ul>
+      </section>
```

Comment:
- [ ] The list is correct and the style is very similar to the Figma template. But this should not be a `section` inside the 'About' section. Kindly remove the `section` tag for better HTML semantics.